codeship/deploy-prod.sh

```diff
@@ -8,5 +8,7 @@ DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 $DIR/build.sh
 
 export AWS_REGION="${AWS_REGION}"
+export CS_PASSWORD="${CS_PASSWORD}"
```

Comment:
Do you happen to know whether a password with a double-quote in it will cause a syntax problem here?

Reply:
I suspect it would be a problem. Unfortunately, I don't know a way around that.